app/gui/src/main/scala/org/bitcoins/gui/dlc/DLCTableView.scala

```diff
@@ -47,7 +47,7 @@ class DLCTableView(model: DLCPaneModel) {
 
     val statusCol = new TableColumn[DLCStatus, String] {
       text = "Status"
-      prefWidth = 150
+      prefWidth = 75
```

Comment:
I'd rather not reduce the size of this one, for `RemoteClaimed` it will be cut off